CoreScriptsRoot/Modules/Chat.lua

```diff
@@ -68,14 +68,12 @@ local allowDisableChatBar = getDisableChatBarSuccess and disableChatBarValue
 while PlayersService.LocalPlayer == nil do PlayersService.ChildAdded:wait() end
 local Player = PlayersService.LocalPlayer
 -- GuiRoot will act as the top-node for parenting GUIs
-local GuiRoot = nil
-if NON_CORESCRIPT_MODE then
-	GuiRoot = Instance.new("ScreenGui")
-	GuiRoot.Name = "RobloxGui"
-	GuiRoot.Parent = Player:WaitForChild('PlayerGui')
-else
-	GuiRoot = CoreGuiService:WaitForChild('RobloxGui')
-end
+local GuiRoot = Instance.new('Frame')
```

Comment:
Had to move it down to https://github.com/SolarCrane/Core-Scripts/blob/vr-keyboard-chat/CoreScriptsRoot/Modules/Chat.lua#L2341 because core-scripts are run before VREnabled  can be set to true. All the functionality will work